### DISCUSSION TITLE
Is it possible to bundle or add Ethernet sniffing interfaces with different port speeds?

### DISCUSSION THREAD

**Discussion by vincentvlk** [Security-Onion-Solutions/securityonion#12413] [Unsupported Versions] [answered]
Hello Community,

Version: 2.4.40
Installation: Standalone
HW: Standard Eth. NICs in PCIe slot, verified working.

According to `https://www.kernel.org/doc/Documentation/networking/bonding.txt` there is not specified any port speed mixing limitation for `balance-rr` bonding mode (used by SO 2.4.40) with different port speeds, but maybe I'm missing something.

Please, is it possible to:

1. Bundle Ethernet sniffing interfaces to `bond0` with different port speeds (e.g. 10G + 100G) during SO installation?

2. Add another Ethernet sniffing interface to `bond0` with different port speeds (e.g. 10G + 100G) after install with `so-monitor-add`?

Thanks in advance for any direction and have a nice day!

**Comment by InfosecGoon** [comment, marked as answer]
You should be able to mix port speeds in the bond0 interface. With that said, sending 110G of raw traffic to one machine is A Lot.
